microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -22,89 +22,105 @@ ms.date: 05/04/2021
    limitations under the License.  -->
 # View page resources
```

Comment:
```suggestion
# View the resource files that make up a webpage
```
Also change:
* "title:" at top of file
* TOC entry
* \microsoft-edge\devtools-guide-chromium\progressive-web-apps\index.md